ovn_cluster.sh

```diff
@@ -517,7 +517,7 @@ function start() {
             sleep 2
             ${RUNC_CMD} exec --detach ${CENTRAL_NAME} bash -c "/run_ovsdb_etcd_sb.sh"
             ${RUNC_CMD} exec --detach ${CENTRAL_NAME} bash -c "/run_ovsdb_etcd_nb.sh"
-            ${RUNC_CMD} exec ${CENTRAL_NAME} ${OVNCTL_PATH} --ovn-manage-ovsdb=no --ovn-northd-log='-vconsole:dbg -vfile:dbg' start_northd
```

Comment:
hm.... any reason you do not like these in dbg? Are they affecting your tests?
I wonder if there is a compromise we can make in keeping them for the folks who added it here.

Reply:
yes they generating tons of logs when run scale test, they are useful for debugging but will hurt scale if you leave them on

Reply:
Okay. I would like the default behavior to remain unchanged, if that is okay with you.
The reason I say that is because many folks use ovn-fake-multinode for learning instead of scaling tests.
Could you please add a new knob to make the original behavior available by default and then use it for our scaling test (maybe in ovn-scale-tests / ovn-heater ) ?

Reply:
oh, my bad. I was under the impression that this is common to both etcd and non-etcd.
In that case, please forget my non-sense request.
lgtm!

Reply:
no worries and thanks for staying on top of my PRs in this repo!!